src/common/tiles/PersonTile/styled.js

```diff
@@ -1,6 +1,6 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
-export const Wrapper = styled.div`
+/* export const Wrapper = styled.div`
```

Comment:
jak najbardziej to można w PersonTile to usunąć, nawet chyba nie było wykorzystywane później, bo to była pierwotna wersja jak nie bylo samej kafelki a strona z kafelkami

Reply:
to mial byc komentarz zwykly, kliknelo mi sie cos innego :)

Reply:
Ok, usunięto :D